src/components/modals/TransactionAdvanceDetails.js

```diff
@@ -56,6 +57,27 @@ class TransactionAdvanceDetailsComponent extends PureComponent<Props> {
     }
   }
 
+  openProveUrl = () => {
+    const { recipientAddress, txid, txSecret } = this.props
+    if (!recipientAddress || !txid || !txSecret) return
```

Comment:
Should at least show a toast so the user knows why nothing happened.

Reply:
Tile isn't visible to press if any of these three fields are null. This conditional was only to satisfy flow